content/articles/matlab-graphical-user-interface/index.md

```diff
@@ -0,0 +1,104 @@
+### GRAPHICAL USER INTERFACE IN MATLAB
+## Introduction
+## Prerequisites
+1. [Matlab](https://www.mathworks.com/downloads/) installed.
+2. proper understanding of [Matlab](https://www.section.io/engineering-education/getting-started-with-matlab/) language.
+
+## How to access the graphical interface in Matlab
+You can access the graphical user interface in Matlab by typing `guide` in Matlab’s workspace.
+![](/engineering-education/matlab-graphical-user-interface/image1.png)
```

Comment:
Specify proper alternative text.

In case, if the image doesn't load, a proper text must show up. So, fill in some text describing the image inside `[]`

Reply:
Do this for all the images